docs/03-dev-howtos/13-Update-configuration-in-s3.md

```diff
@@ -12,35 +12,35 @@ To add or update a configuration item you need to:
 aws s3 ls --profile=frontend s3://aws-frontend-store/config/
 ```
 
-look for the most recent version ( the `v14` part ):
+look for the most recent version ( the `v17` part ):
```

Comment:
It is definitely not required to bump those. 
I just find it easier when I have to update the config. I just have to copy paste without changing the number. 
Anyway the whole process is cumbersome. Hopefully the new devtools team will come up with a nice config tool :crossed_fingers: